hacks/update-jenkins-plugins/collect-jenkins-plugins.sh

```diff
@@ -112,7 +112,7 @@ get_plugin() {
     wget -O "${tmp_hpi_file}" "${plugin_url}" 2> /dev/null
     if [ "$?" != "0" ]; then
         echo "Error downloading ${plugin}; exiting"
-        exit 1
+        return 1
```

Comment:
relate to https://github.com/openshift/aos-cd-jobs/pull/3468, job can continue if latest dir not exist